pkgs/development/libraries/SDL2_gfx/default.nix

```diff
@@ -9,6 +9,8 @@ stdenv.mkDerivation rec {
     sha256 = "16jrijzdp095qf416zvj9gs2fqqn6zkyvlxs5xqybd0ip37cp6yn";
   };
 
+  nativeBuildInputs = stdenv.lib.optional stdenv.isDarwin darwin.libobjc;
```

Comment:
Are you sure this should be in `nativeBuildInputs`?

Reply:
My bad. Probably `libobjc` should be in `buildInputs`.